nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/test/java/org/apache/nifi/processors/standard/TestInvokeHttpTwoWaySSL.java

```diff
@@ -51,4 +59,27 @@ public static void beforeClass() throws Exception {
         url = server.getSecureUrl();
     }
 
+    @AfterClass
+    public static void afterClass() throws Exception {
+        if(server != null) {
```

Comment:
Recommend checking the spacing here:
```suggestion
        if (server != null) {
```